### PR TITLE
More mods and updates!

### PR DESCRIPTION
**Pull request by Banane9**
Added:
* New Version of EditorTabbing
* Add hantabaru1014 as Author to EditorTabbing for PR
* New Version of UserJoinLeaveNotifications
* New Mod CorrectGizmoScaling
* New Mod BoundedUIX
* New Mod ComponentAttacherSearch

Also added myself as an Author on Art's color drop mod for the PR 🤔

### CODE REVIEW

**Comment by Banane9** [comment]
Also add PrecisionBlendShapeBaking

**Comment by XDelta** [comment]
Consider using branches on your fork to make PRs that are only one mod each. Makes it easier to review one at a time versus needing to do 5+ and any issue on one holds up any additional entries.

**Comment by Banane9** [comment]
I'll try to be faster with adding them to the manifest 😅

**Comment by XDelta** [comment]
All good, looking over these at the moment.

**Comment by Banane9** [comment]
I hope the transpiler ones are simple enough 👍

File: manifest.json
Comment (XDelta) [inline manifest.json:3900]: Wrong Hash
Expected:
ffb3dc6aaab2e18795a9b8200f01ae3b370087d22d754d47f24b3daa515e35e5
Got:
6a0a7e225307add0fb4a704edf92290fef1a406f5ecb8a111e05fa97993e3a8d

**Review by XDelta — CHANGES_REQUESTED** [review]

File: manifest.json
Comment (XDelta) [inline manifest.json:3585]: +1

File: manifest.json
Comment (XDelta) [inline manifest.json:3608]: +1

File: manifest.json
Comment (XDelta) [inline manifest.json:3643]: +1

File: manifest.json
Comment (XDelta) [inline manifest.json:3779]: +1

File: manifest.json
Comment (XDelta) [inline manifest.json:3885]: May want to either include some of this in the description or add your [ComponentAttacherFavorites](https://github.com/Banane9/NeosComponentAttacherFavorites) as a requirement.

 "Requires my forked version of [ComponentAttacherFavorites](https://github.com/Banane9/NeosComponentAttacherFavorites) to work ideally.
Workaround is to make the this mod load before the other."

File: manifest.json
Comment (XDelta) [inline manifest.json:3780]: +1

File: manifest.json
Comment (XDelta) [inline manifest.json:3609]: +1

File: manifest.json
Comment (Banane9) [reply manifest.json:3900]: hmmm, I forgot to update the hash from the one I copied... but also it wasn't the hash of the release build - weird.
I updated the hash and file, so it should match now

File: manifest.json
Comment (Banane9) [reply manifest.json]: Added my favorites version to the list as well with a note in the description of the search about which one to use / what to do.

File: manifest.json
Comment (XDelta) [inline manifest.json:3860]: +1

File: manifest.json
Comment (XDelta) [inline manifest.json:3906]: +1

File: manifest.json
Comment (XDelta) [inline manifest.json:3883]: +1

File: manifest.json
Comment (XDelta) [inline manifest.json:3837]: +1

File: manifest.json
Comment (XDelta) [inline manifest.json:1405]: It is up to @art0007i if they'd like to add you as an author on their mod. You can wait for a response from them or remove this change and make it a separate PR so that this one can proceed to be merged.

**Review by XDelta — CHANGES_REQUESTED** [review]

**Comment by XDelta** [comment]
Everything else on this PR looks good. Can be merged after the one thing above is resolved.

File: manifest.json
Comment (art0007i) [reply manifest.json:1405]: yeah im cool with it

Comment (XDelta) [reply manifest.json:1405]: Alright

**Review by XDelta — APPROVED** [review]

**Pull request merged**
